lighthouse-cli/test/smokehouse/frontends/smokehouse-bin.js

```diff
@@ -130,21 +147,37 @@ async function begin() {
   const requestedTestIds = argv._;
   const allTestDefns = require(testDefnPath);
   const invertMatch = argv.invertMatch;
-  const testDefns = getDefinitionsToRun(allTestDefns, requestedTestIds, {invertMatch});
+  const onlyUrls = argv.onlyUrls;
+  const testDefns = getDefinitionsToRun(allTestDefns, requestedTestIds, {invertMatch, onlyUrls});
 
   const options = {jobs, retries, isDebug: argv.debug, lighthouseRunner};
 
-  let isPassing;
+  let testResults;
   try {
     server.listen(10200, 'localhost');
     serverForOffline.listen(10503, 'localhost');
-    isPassing = (await runSmokehouse(testDefns, options)).success;
+    testResults = await runSmokehouse(testDefns, options);
   } finally {
     await server.close();
     await serverForOffline.close();
   }
 
-  const exitCode = isPassing ? 0 : 1;
+  const failingUrls = testResults.testResults
```

Comment:
should the variable be something like `smokehouseResult`? `testResults.testResults` twitches my nose a bit :)